components/_global-footer.scss

```diff
@@ -463,31 +448,37 @@
 
 		// items in Wikia section
 		&__wikia-section {
+			padding-bottom: $section-vertical-padding;
+
 			&.wds-is-company-overview {
-				height: $desktop-row-top-height;
 				margin-right: 50px;
+				min-height: $desktop-row-top-height;
+				order: 1;
 				width: $desktop-second-column-width;
 			}
 
 			&.wds-is-site-overview {
-				height: $desktop-row-bottom-height;
 				margin-right: 50px;
+				order: 4;
 				width: $desktop-second-column-width;
 			}
 
 			&.wds-is-community {
-				height: $desktop-row-top-height;
+				min-height: $desktop-row-top-height;
+				order: 2;
 				width: $desktop-third-column-width;
 			}
 
 			&.wds-is-create-wiki {
-				height: $desktop-row-bottom-height;
 				display: block;
+				order: 5;
 				width: $desktop-third-column-width;
 			}
 
 			&.wds-is-community-apps {
-				height: $desktop-row-top-height;
+				margin-right: 0;
+				min-height: $desktop-row-top-height;
+				order: 3;
```

Comment:
On mobile we display sections one under another. On desktop I decided to take another approach and display them one next to another and break row after 3rd.

The reason why I decided to change flex-direction:
* Previously on desktop we tried to emulate the same flow by changing flex-direction to columns, then we had to limit height of the container to force flex to wrap column. Finally we end up with static height containers.
* css column doesn't allow to manipulate column widths
* floats won't work, as content is in different order

Other option is to add additional HTML markup only for styling columns, which I personally don't like.